akka-http-core/src/main/java/akka/http/javadsl/model/sse/ServerSentEvent.java

```diff
@@ -18,18 +18,19 @@
 
 import java.util.Optional;
 import java.util.OptionalInt;
+
+import scala.jdk.javaapi.OptionConverters;
 import scala.Option;
-import static scala.compat.java8.OptionConverters.toScala;
 
 /**
  * Representation of a server-sent event. According to the specification, an empty data field
  * designates an event which is to be ignored which is useful for heartbeats.
  */
 public abstract class ServerSentEvent {
 
-    private static final Option<String> stringNone =  toScala(Optional.empty());
+    private static final Option<String> stringNone = OptionConverters.toScala(Optional.empty());
 
-    private static final Option<Object> intNone = toScala(OptionalInt.empty());
+    private static final Option<Object> intNone = OptionConverters.toScala(OptionalInt.empty()).map(emptyIntOption -> (Object)emptyIntOption);
```

Comment:
These two are really weird to start with, `Option.empty()` should be fine for both?